privacy_evaluator/models/tf/cnn.py

```diff
@@ -11,10 +11,12 @@
 
 
 def ConvNet(
-    num_classes: int = 2, input_shape: Tuple[int, ...] = (1, 28, 28)
+    num_classes: int = 2,
+    input_shape: Tuple[int, ...] = (1, 28, 28),
+    num_channels: Tuple[int, ...] = (1, 16, 32, 64)
 ) -> keras.Model:
+
```

Comment:
I think it is better to leave the default values out since input_shape is determined automatically and num_channels is hard-coded anyways for CIFAR10 in line 20. Or should we rather delete the hard-coded value for CIFAR10? Nevertheless, I don't see why it is beneficial to have these default values here. Was there a specific reason why you added them?